File: DotNet/Framework/AutomationFramework/Helpers/Drivers/IDriver.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Remote;

namespace Helpers.Drivers
{
    public interface IDriver
    {
        public IWebDriver GetDriver();
        public void SetDriver();
        public void MaximizeDriver();
        public void MinimizeDriver();
        public void NavigateURL(string url);
        public void CloseDriver();
    }
}

File: Java/AutomationFramework/src/com/app/helpers/drivers/IDriver.java
package com.app.helpers.drivers;

import org.openqa.selenium.WebDriver;

public interface IDriver {
    WebDriver GetDriver();
    void SetDriver();
    void MaximizeDriver();
    void MinimizeDriver();
    void NavigateURL(String url);
    void CloseDriver();

    void QuitDriver();
}
